web/src/app/modules/sugarloaf/components/smart/container/container.component.spec.ts

```diff
@@ -108,4 +114,77 @@ describe('AppComponent', () => {
       }
     ));
   });
+
+  xdescribe('with history updates', () => {
```

Comment:
Is the x on "xdescribe" necessary any more?

Reply:
Yes, the plan was to address this in the future as this works as focused test but not with whole suite. There is a similar issue with tests for resource viewer and cytoscape components as well. As of now, I have removed flaky tests related to this PR and they are now pushed here: `https://github.com/xtreme-vikram-yadav/octant/tree/route-history-with-flakes` which i plan to address in the future tech debt week.